lang/en.yml

```diff
@@ -1,4 +1,6 @@
 en:
+  DevelopmentAdmin:
+    php.SilverStripe\Dev\DevelopmentAdminPERMISSIONS_CATEGORY: 'Dev permissions'
```

Comment:
This doesn't look right

There's a missing dot [here](https://github.com/silverstripe/silverstripe-framework/blob/5/src/Dev/DevelopmentAdmin.php#L285) - could you please add it and re-run translations

Reply:
Good catch. Fixed, and raised [an issue](https://github.com/silverstripe/silverstripe-standards/issues/5) to make a phpstan rule to catch these in CI in the future.